schemas/2015-06-01/Microsoft.Cdn.json

```diff
@@ -0,0 +1,401 @@
+          "sku"
+        ]
+      },
+      "resources": {
+        "type": "array",
+        "items": {
+          "allOf": [
+            {
+              "$ref": "http://schema.management.azure.com/schemas/2015-01-01/deploymentTemplate.json#/definitions/resourceBase"
+            },
+            {
+              "oneOf": [
+                {
+                  "$ref": "http://schema.management.azure.com/schemas/2014-10-01-preview/Microsoft.Authorization.json#/resourceDefinitions/roleAssignments"
+                },
+                {
+                  "$ref": "http://schema.management.azure.com/schemas/2015-01-01/Microsoft.Authorization.json#/resourceDefinitions/locks"
+                },
+                {
+                  "$ref": "http://schema.management.azure.com/schemas/2015-01-01/Microsoft.Resources.json#/resourceDefinitions/links"
+                },
+                {
+                  "$ref": "#/resourceDefinitions/endpointChild"
+                }
+              ]
+            }
+          ]
+        },
+        "description": "Microsoft.Cdn/profiles: Child resources in a CDN profile."
+      },
+      "required": [
+        "type",
+        "apiVersion",
+        "properties",
+        "location"
+      ]
+    },
+    "endpoints": {
+      "type": "object",
+      "description": "The CDN endpoint",
+      "properties": {
+        "type": {
+          "enum": [
+            "Microsoft.Cdn/profiles/endpoints"
+          ]
+        },
+        "apiVersion": {
+          "enum": [
+            "2015-06-01"
+          ]
+        },
+        "properties": {
+          "$ref": "#/definitions/endpointCommon"
+        }
+      },
+      "resources": {
+        "$ref": "#/definitions/endpointResourcesCommon"
+      },
+      "required": [
+        "type",
+        "apiVersion",
+        "properties",
+        "location"
+      ]
+    },
+    "endpointChild": {
+      "type": "object",
+      "description": "The CDN endpoint",
+      "properties": {
+        "type": {
+          "enum": [
+            "endpoints"
+          ]
+        },
+        "apiVersion": {
+          "enum": [
+            "2015-06-01"
+          ]
+        },
+        "properties": {
+          "$ref": "#/definitions/endpointCommon"
+        },
+        "required": [
+          "origins"
+        ]
+      },
+      "resources": {
+        "$ref": "#/definitions/endpointResourcesCommon"
+      },
+      "required": [
+        "type",
+        "apiVersion",
+        "properties",
+        "location"
+      ]
+    },
+    "origins": {
+      "type": "object",
+      "description": "The CDN origin",
+      "properties": {
+        "type": {
+          "enum": [
+            "Microsoft.Cdn/profiles/endpoints/origins"
+          ]
+        },
+        "apiVersion": {
+          "enum": [
+            "2015-06-01"
+          ]
+        },
+        "properties": {
+          "$ref": "#/definitions/originCommon"
+        }
+      },
+      "required": [
```

Comment:
No, it's not required.
https://msdn.microsoft.com/en-us/library/azure/mt634429.aspx